src/Storage/Query/Handler/OrderHandler.php

```diff
@@ -17,12 +17,41 @@ class OrderHandler
      */
     public function __invoke(QueryInterface $query, $order)
     {
-        if (strpos($order, '-') === 0) {
-            $direction = 'DESC';
-            $order = substr($order, 1);
-        } else {
-            $direction = null;
+        $separatedOrders = $this->getOrderBys($order);
+
+        foreach ($separatedOrders as $order) {
+            $order = trim($order);
+            if (strpos($order, '-') === 0) {
+                $direction = 'DESC';
+                $order = substr($order, 1);
+            } else {
+                $direction = null;
+            }
+            $query->getQueryBuilder()->orderBy($order, $direction);
         }
-        $query->getQueryBuilder()->orderBy($order, $direction);
+
+    }
+
+    /**
+     * @param $order
+     * @return array
+     */
+    protected function getOrderBys($order)
+    {
+        $separatedOrders[] = $order;
+
+        if ($this->isMultiOrderQuery($order))
+            $separatedOrders = explode(",", $order);
+
+        return $separatedOrders;
+    }
+
+    /**
+     * @param $order
+     * @return bool
+     */
+    protected function isMultiOrderQuery($order)
+    {
+        return ( strpos($order, ',') !== FALSE ? TRUE : FALSE );
```

Comment:
Per [PSR-2](https://github.com/php-fig/fig-standards/blob/master/accepted%2FPSR-2-coding-style-guide.md#25-keywords-and-truefalsenull) the booleans should be lower case.